src/braket/default_simulator/local_execution_manager.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from typing import Union
+
+from braket.task_result import (
+    AnalogHamiltonianSimulationTaskResult,
+    AnnealingTaskResult,
+    GateModelTaskResult,
+)
+
+from braket.simulator.quantum_task import ExecutionManager
+
+
+class LocalExecutionManager(ExecutionManager):
+    """A task containing the results of a local simulation.
+
+    Since this class is instantiated with the results, cancel() and run_async() are unsupported.
```

Comment:
This isn't instantiated with results, right? Also, `run_async` isn't an `ExecutionManager` method

Reply:
I forgot to update documentation here. My bad,  Let me update it.